pocket/chain.json

```diff
@@ -0,0 +1,122 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "pocket",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://pocket.network/",
+  "pretty_name": "Pocket Network",
+  "chain_type": "cosmos",
+  "chain_id": "pocket",
+  "bech32_prefix": "pokt",
+  "daemon_name": "pocketd",
+  "node_home": "$HOME/.pocket",
+  "key_algos": ["secp256k1"],
+  "slip44": 118,
+  "description": "Pocket Network is a decentralized infrastructure protocol that incentivizes a global network of node operators to provide reliable and cost-effective data access for applications across multiple platforms, including blockchains, AI services, and other APIs.",
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "upokt",
+        "fixed_min_gas_price": 0.001,
+        "low_gas_price": 0.001,
+        "average_gas_price": 0.002,
+        "high_gas_price": 0.0025
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "upokt"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/pokt-network/poktroll",
+    "recommended_version": "v0.1.1",
+    "compatible_versions": ["v0.1.1"],
+    "binaries": {
+      "linux/amd64": "https://github.com/pokt-network/poktroll/releases/download/v0.1.1/pocket_linux_amd64.tar.gz",
+      "linux/arm64": "https://github.com/pokt-network/poktroll/releases/download/v0.1.1/pocket_linux_arm64.tar.gz",
+      "darwin/amd64": "https://github.com/pokt-network/poktroll/releases/download/v0.1.1/pocket_darwin_amd64.tar.gz",
+      "darwin/arm64": "https://github.com/pokt-network/poktroll/releases/download/v0.1.1/pocket_darwin_arm64.tar.gz"
+    },
+    "consensus": {
+      "type": "cometbft",
+      "version": "v0.38.12"
+    },
+    "genesis": {
+      "name": "v0.1.1",
+      "genesis_url": "https://raw.githubusercontent.com/pokt-network/pocket-network-genesis/refs/heads/master/shannon/mainnet/genesis.json"
+    },
+    "sdk": {
+      "type": "cosmos",
+      "version": "v0.50.13"
+    },
+    "ibc": {
+      "type": "go",
+      "version": "v8.7.0"
+    },
+    "cosmwasm": {
+      "enabled": false
+    }
+  },
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/pocket/images/pokt-icon.png",
+      "theme": {
+        "circle": true,
+        "primary_color_hex": "#0C3B9D"
+      }
+    },
+    {
+      "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/pocket/images/pokt-icon.svg",
+      "theme": {
+        "circle": true,
+        "primary_color_hex": "#0C3B9D"
+      }
+    }
+  ],
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/pocket/images/pokt-icon.png",
```

Comment:
For most chains the png and svg are visually identical, and can be grouped together, but since in this case the png and svg are different, they cannot both be the default, so you can only have either the png or svg here (whichever you choose should also be the first in the list of images)